### ISSUE TITLE
ENH: Scipy writing 24-bit signed wav file

### ISSUE THREAD

**Issue by vong0** [scipy/scipy#16710]
### Is your feature request related to a problem? Please describe.

I know that scipy has support to read 24-bit signed wav files (and that it reads it into a int32 data type).

Does it also support writing 24-bit signed wav files? I don't see any reference in the scipy.io.wavfile.write docs.

If it does not support writing 24-bit wav files, is there a workaround to do so?

**Comment by rkern** [comment]
No, it's not supported. There isn't particularly a workaround, but you are free to [cannibalize the implementation](https://github.com/scipy/scipy/blob/f662651cbd8191855c66185f66f20d7a2fab301b/scipy/io/wavfile.py#L710) for parts.
